packages/client/src/pages/home/action-group/post-dialog.tsx

```diff
@@ -81,7 +82,7 @@ const PostDialog = ({ open, onClose }: PostDialogProps) => {
         //TODO use real user id once login is set up
         userId: "60f138a5910aef4526a82182",
       }),
-    );
+    ).then(() => dispatch(getPostsByFilter(postState)));

```

Comment:
maybe we could handle this client side instead (since we have the new post + current posts). maybe maybe maybe maybe

Reply:
or maybe we don't do it at all, users just have to refresh to see their post nyeheheh

Reply:
post-slice always appends the created post so I didn't want the dirty posts to appear unexpectedly 😱 maybe I should just check it there instead of sending another request hmMMMmM 

I ALSO NEED TO FIX THAT TODO NOOOOOOO

Reply:
actually since there's probably only one workflow to create posts, the createPosts actions could be responsible for filtering too instead of just pushing. Or alternatively, the endpoint could return back a filtered list instead of a single post (but that feels more jank)

Reply:
hmmMMmM calling getPostsByFilter when createPost succeeds just so it's not doing it in a far away land like the post dialog